src/engraving/api/v1/elements.h

```diff
@@ -1220,6 +1221,15 @@ class Staff : public ScoreElement
 
     Part* part();
     /// \endcond
+
+    /**
+     * Returns the clef type in effect at the given tick.
```

Comment:
```suggestion 
     /// Returns the clef type in effect at the given tick.
```
Probably better to use a comment style that matches the rest of the file.